polars/polars-io/src/parquet/read_impl.rs

```diff
@@ -168,7 +168,7 @@ fn rg_to_dfs_par(
     remaining_rows: &mut usize,
     file_metadata: &FileMetaData,
     schema: &ArrowSchema,
-    predicate: Option<Arc<dyn PhysicalIoExpr>>,
+    predicate: &Option<Arc<dyn PhysicalIoExpr>>,
```

Comment:
I think the ref should be inside the option.

Reply:
Changed.